tests/test_formatter.py

```diff
@@ -4,18 +4,20 @@
 from todoman import ui
 
 DATE_FORMAT = "%d-%m-%y"
+TIME_FORMAT = "%H:%M"
 
 
-def test_format_date():
-    """
-    Tests the format_date function in todoman.ui.TodoFormatter
-    """
-    formatter = ui.TodoFormatter(DATE_FORMAT)
+def test_human_dates():
+    formatter = ui.TodoFormatter(DATE_FORMAT, TIME_FORMAT, ' ')
     today = datetime.now()
     tomorrow = today + timedelta(days=1)
+    tomorrow_12am = datetime(tomorrow.year, tomorrow.month, tomorrow.day,
+                             12, 0)
     any_day = today + timedelta(days=randrange(2, 8))
 
-    assert formatter.format_date("") == "        "
-    assert formatter.format_date(today) == "   Today"
-    assert formatter.format_date(tomorrow) == "Tomorrow"
-    assert formatter.format_date(any_day) == any_day.strftime(DATE_FORMAT)
+    assert formatter.format_datetime("") == "              "
+    assert "Today" in formatter.format_datetime(today.date())
```

Comment:
I prefer the version with the string with spaces (eg: `"    Today"`), because it also checks that we don't break column width.